pkg/acceptance/bettertestspoc/assert/resourceshowoutputassert/stream_show_output_ext.go

```diff
@@ -0,0 +1,29 @@
+// Code generated by assertions generator; DO NOT EDIT.
+
+package resourceshowoutputassert
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/bettertestspoc/assert"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+)
+
+func (s *StreamShowOutputAssert) HasCreatedOnNotEmpty() *StreamShowOutputAssert {
+	s.AddAssertion(assert.ResourceShowOutputValuePresent("created_on"))
+	return s
+}
+
+func (s *StreamShowOutputAssert) HasStaleAfterNotEmpty() *StreamShowOutputAssert {
+	s.AddAssertion(assert.ResourceShowOutputValuePresent("stale_after"))
+	return s
+}
+
+func (s *StreamShowOutputAssert) HasBaseTables(ids []sdk.SchemaObjectIdentifier) *StreamShowOutputAssert {
+	s.AddAssertion(assert.ResourceShowOutputValueSet("base_tables.#", strconv.FormatInt(int64(len(ids)), 10)))
+	for i := range ids {
+		s.AddAssertion(assert.ResourceShowOutputValueSet(fmt.Sprintf("base_tables.%d", i), ids[i].FullyQualifiedName()))
```

Comment:
Imo we should add a new assert that would assert length, then items, and it should be reliant on order of saved items. I created a ticket for it and added an item to the list of assert improvements.

Reply:
let's add it to the readme?

Reply:
I added it in my pr